newscoop/src/Newscoop/GimmeBundle/Form/Type/ArticleType.php

```diff
@@ -27,16 +26,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         ->add('name', 'text', array(
             'required' => true,
         ))
-        ->add('language', 'text', array(
+        ->add('languageId', 'text', array(
```

Comment:
this is bc break for api ;/ we can't merge it to 4.4. How about not changing form field names but just providing some kind description for those parameters? 

```For Form Types, you can add an extra option named description on each field:```
https://github.com/nelmio/NelmioApiDocBundle/blob/master/Resources/doc/index.md#the-apidoc-annotation